lightning/src/ln/channelmanager.rs

```diff
@@ -289,15 +289,15 @@ const ERR: () = "You need at least 32 bit pointers (well, usize, but we'll assum
 /// lifetimes). Other times you can afford a reference, which is more efficient, in which case
 /// SimpleRefChannelManager is the more appropriate type. Defining these type aliases prevents
 /// issues such as overly long function definitions.
-pub type SimpleArcChannelManager<M, T> = Arc<ChannelManager<InMemoryChannelKeys, Arc<M>, Arc<T>>>;
+pub type SimpleArcChannelManager<M, T, K> = Arc<ChannelManager<InMemoryChannelKeys, Arc<M>, Arc<T>, Arc<K>>>;
```

Comment:
Should probably be KeysManager instead of a new template type. Same below. Also should maybe get a documentation update if you think it makes sense.